corehq/apps/app_manager/suite_xml/sections/entries.py

```diff
@@ -653,19 +654,20 @@ def get_load_case_from_fixture_datums(self, action, target_module, form):
                 requires_selection=True,
                 action=action,
             ))
-        datums.append(FormDatumMeta(
-            datum=SessionDatum(
-                id=load_case_from_fixture.fixture_tag,
-                nodeset=load_case_from_fixture.fixture_nodeset,
-                value=load_case_from_fixture.fixture_variable,
-                detail_select=self.details_helper.get_detail_id_safe(target_module, 'case_short'),
-                detail_confirm=self.details_helper.get_detail_id_safe(target_module, 'case_long'),
-                autoselect=load_case_from_fixture.auto_select_fixture,
-            ),
-            case_type=action.case_type,
-            requires_selection=True,
-            action=action,
-        ))
+        if load_case_from_fixture.fixture_tag:
```

Comment:
can we not validate this prior to build or on save?

Reply:
I was thinking of validating it on save, but the use-case which illuminated the issue does not need to have to fields filled out as they're using the feature a bit differently as I understand it; I'm not too clued up on what exactly they're trying to do by using it this way (let me quickly find out), just know that they don't fill in the fields, so I don't want to force the front-end to take this input.

Reply:
@snopoke The use-case requires the Fixture fields to be empty since they want to be able to add an arbitrary datum element to the form. Validating that the fields be populated before save might impede this use-case, that's why I though of just skipping the creation of the datum with the invalid data.

Reply:
Interesting, thanks for the follow up.